### ISSUE TITLE
Data breach problem

### ISSUE THREAD

**Issue by Yucioiooo** [zbliu98/GRAM-ODE#1]

**Comment by zbliu98** [comment]
Yes, we calculated mean and std for train, valid, and test set. But in each train(eval/test) set, we only use its mean std. So, I don't think it has a data breach problem.

**Comment by Yucioiooo** [comment]
Sorry, at some point, shouldn't we don't know anything about the validation set and the test set?

**Comment by zbliu98** [comment]
Yes, you are right. Calculating mean and std on train/eval/test in the beginning is only for convenience.
